Bioinfo_joint/HW4_modules/natls.py

```diff
@@ -0,0 +1,110 @@
+def dict_comp(nucleotide: str) -> str:
+    complement = dict(
+        [
+            ("a", "t"),
+            ("A", "T"),
+            ("c", "g"),
+            ("C", "G"),
+            ("g", "c"),
+            ("G", "C"),
+            ("t", "a"),
+            ("T", "A"),
+            ("u", "a"),
+            ("U", "A"),
+            ("0", "0"),
+        ]
+    )
+    return complement[nucleotide]
+
+
+def dict_trans(nucleotide: str) -> str:

```

Comment:
Аналогично функции `dict_comp`, название можно изменить. Например на
```suggestion
def get_transcript(nucleotide: str) -> str:
```